src/binance/bnbPlugin.js

```diff
@@ -16,6 +16,7 @@ import {
   type EdgeIo,
   type EdgeMetaToken,
   type EdgeParsedUri,
+  type EdgeParseUriOptions,
```

Comment:
In the core, make this type a cleaner that you can use throughout the plug-in to validate the options